### PR TITLE
WIP: Support emulator-10-x86_64

### PR DESCRIPTION
**Pull request by lissyx**

### CODE REVIEW

**Comment by lissyx** [comment]
@seinlin can we merge? it includes the commit from gabriele's PR as well, so should merge his PR first then I'll rebase and we can merge

File: run-gdb.sh [hunk 1]
```diff
@@ -37,34 +41,20 @@ echo "ADB Location: " $ADB
...
-        TARGET_ARCH=x86
-        TARGET_TRIPLE=x86_64-linux-android
+    aosp_x86_64)
+        BINSUFFIX=64
+        DEVICE_NAME=generic_x86_64
         ;;
     *)
-        TARGET_ARCH=arm
-        TARGET_TRIPLE=arm-linux-androideabi
+        DEVICE_NAME=generic
```
Comment (seinlin) [inline run-gdb.sh]: nit: it will be better to keep *) no change then we can use this for device debug as well.

**Comment by seinlin** [comment]
@lissyx , It is fine to merge it once we addressed the nit as commented. Thanks!

File: run-gdb.sh (hunk 1 above)
Comment (lissyx) [reply run-gdb.sh]: Unsure about your request regarding the changes made by @gabrielesvelto, but I'm reverting this one

**Comment by seinlin** [comment]
Merged in another PR.

**Pull request closed without merging**